docs/operational_guide/namespace_configuration.md

```diff
@@ -10,6 +10,12 @@ The operations below include sample cURLs, but you can always review the API doc
 
 `http://<M3_COORDINATOR_HOST_NAME>:<CONFIGURED_PORT(default 7201)>/api/v1/openapi` or our [online API documentation](https://m3db.io/openapi/).
 
+Additionally, you can use the below headers in the namespace operations: 
```

Comment:
Nit: `... the following headers can be used ...` per our [style guide](https://docs.m3db.io/misc/writing_docs/) method of avoiding first/second person pronouns